### ISSUE TITLE
Broken symlink causes unhandled exception

### ISSUE THREAD

**Issue by KoleckOLP** [ppy/osu#9717]
**Describe the bug:** 
I wasn't able to to install and run, osu!lazer, than I tryed to build from the source. And I got an error related to my Roaming/osu so I looked there and I had a symlink to a folder than nolonger existed.
(the issue is broken osu folder symlink in %appdata% crashes osu!lazer)

**Screenshots or videos showing encountered issue:** 
(I don't have a screenshot, I just removed the symlink and it started to work.)

**osu!lazer version:** 
2020.725.0

**Logs:**
[database.log](https://github.com/ppy/osu/files/4996371/database.log)
[performance.log](https://github.com/ppy/osu/files/4996372/performance.log)
[runtime.log](https://github.com/ppy/osu/files/4996373/runtime.log)
[updater.log](https://github.com/ppy/osu/files/4996374/updater.log)

**Comment by bdach** [comment]
Did you check *what* the symlink was and where it lead to? I am not aware of lazer creating any symlinks during its runtime, and due to that I'm not sure what to do with this report.

**Comment by KoleckOLP** [comment]
the symlink was probably a leftover from when I followed this
https://twitter.com/ppy/status/1234740619538386945 (peppy's tutorial to move the folder to another drive before the game could do that.)

I was just really surprised that I can't install and run osu!lazer. The game just did not open no error nothing, I had to try to compile from source to find out that I had a broken symlink.

I just don't want other people to be suck with non working install of lazer. with no error teling them what's wrong.
